types/core/index.d.ts

```diff
@@ -293,6 +298,9 @@ export declare class Chart<
   hide(datasetIndex: number): void;
   show(datasetIndex: number): void;
 
+  getActiveElements(): ChartActiveElement[];
```

Comment:
Result contains also `element` which is probably usable in some situations, so it should be included in the result type.